crates/burn-dataset/src/source/huggingface/downloader.rs

```diff
@@ -74,6 +75,7 @@ impl HuggingfaceDatasetLoader {
             base_dir: None,
             huggingface_token: None,
             huggingface_cache_dir: None,
+            trust_remote_code: true,
```

Comment:
We shouldn't enable this by default. It's a big security hole. It's not enabled by default and we should let the users decide.